app/views/splash_screen/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<section class="d-flex flex-column align-items-center justify-content-center" style="height:100vh;">
+  <div>
+    <h1>Budget App</h1>
+  </div>
+  <div class="d-flex flex-column gap-4 mt-5">
+    <% if user_signed_in?%>
+      <div>
+        <%= link_to "Categories", categories_path, class: "btn btn-primary w-100
```

Comment:
- [ ] This is the link that shows up when the user logs in. Kindly note that the categories index page is what should be presented after logging in. The user does not need to click any link after logging in.

  ![dieoe](https://user-images.githubusercontent.com/54848363/214601434-460bc29f-1e58-4035-acec-e1fcc9e1711b.jpg)